keycloak/keycloak-operator
# Keycloak operator: add accessTokenLifespanForImplicitFlow to KeycloakRealm

## JIRA ID
[KEYCLOAK-17821](https://issues.redhat.com/browse/KEYCLOAK-17821)


## Verification Steps

* Create a Keycloak Realm with accessTokenLifespanForImplicitFlow set
* Check in the keycloak admin that the accessTokenLifespanForImplicitFlow is set correctly under "access tokens"

## Checklist:
